src/graphics/device.js

```diff
@@ -776,6 +808,7 @@ Object.assign(GraphicsDevice.prototype, {
 
         this.extDebugRendererInfo = getExtension('WEBGL_debug_renderer_info');
         this.extTextureFloatLinear = getExtension("OES_texture_float_linear");
+        this.extFloatBlend = getExtension("EXT_float_blend");
```

Comment:
Is this required for texture based morphing? It's not available on iOS.

Reply:
float blending is only used when float render targets are available, so not used on iOS - there a half float format is used instead.